grace/base.py

```diff
@@ -37,27 +37,48 @@ class Annotation(enum.IntEnum):
 
 @dataclasses.dataclass
 class Prediction:
-    """Value of predicted label (argmax) of all
-    normalised class probabilities (softmax).
+    """Prediction dataclass all normalised softmax class probabilities.
 
     Parameters
     ----------
-    LABEL : int
-        Predicted class label (argmax)
-    PROB_TN : float
-        Probability of true negative detection (softmax)
-    PROB_TP : float
-        Probability of true positive detection (softmax)
+    softmax_probabs : npt.NDArray
+        Array or normalised softmax probs as predicted by classifier.
+
+    Methods
+    -------
+    label : self -> Annotation
```

Comment:
```suggestion
    label : Annotation
```

similar for the others below